xgi/drawing/layout.py

```diff
@@ -295,17 +295,17 @@ def weighted_barycenter_spring_layout(H, return_phantom_graph=False, seed=None):
         return pos
 
 
-def pca_transform(pos, theta, degrees=True):
+def pca_transform(pos, theta=0, degrees=True):
     """Transforms the positions of the nodes based on the
     principal components.
 
     Parameters
     ----------
     pos : dict of numpy arrays
         The output from any layout function
-    theta : float
+    theta : float, optional
         The angle between the horizontal axis and the principal axis
-        measured counterclockwise.
+        measured counterclockwise, by default 0.
     degrees : bool, optional
         Whether the angle specified is in degrees, by default True
```

Comment:
"If False, the angle is radians" I guess

Reply:
Fixed.